docs/_components/creating-a-project.rst

```diff
@@ -1,27 +0,0 @@
-* **Create a project**
-
-  Start off by visiting https://cloud.google.com/console
-  and click on the big red button
-  that says "Create Project".
-
-* **Choose a name**
-
-  In the box that says "name",
-  choose something friendly.
-  This is going to be the *human-readable* name
-  for your project.
-
-* **Choose an ID**
-
-  In the box that says "ID",
-  choose something unique
-  (hyphens are OK).
-  I typically choose a project name
-  that starts with my initials,
-  then a hyphen,
-  then a unique identifier for the work I'm doing.
-  For this example,
-  you might choose ``<initials>-quickstart``.
-
-Then click OK
-(give it a second to create your project).
```

Comment:
We haven't published this document for over 6 months. There is an equivalent somewhere in `gcloud-common` we should look at.